data/remnant missions.txt

```diff
@@ -510,7 +510,7 @@ mission "Remnant: Void Sprites 3"
 				`	"Sorry, that's more than I'm willing to commit to."`
 					decline
 
-			`	"Excellent," he says. "The ship should be available for purchase in the shipyard on Viminal."`
+			`	"Excellent," he says. "The ship should be available for purchase in the shipyard on Viminal." He pauses and looks thoughtful, "Just be careful with the shields. It is an antique that uses an old hard-shell system instead of the skin generators that were becoming common at the time of the exodus. It has compensators that will bend the shield to match the ship and any synchronized equipment such as the pressure suit, but it isn't smart enough to adapt to new obstacles." He pauses, then continues. "Just make sure nothing valuable is near the hull when the shields activate - it will either be sheared in two or break the shields."`
```

Comment:
I'd reverse the order of the information here; have the explanation first and the technobabble second instead of the other way around. And keep the information about the suit separate from the information about the mechanics of the shields.
e.g. -> `"This ship is an antique, so be careful with the shields. They are projected around the ship and are unable to adapt to new obstacles. Hard-shell shields, I believe they were called at the time of the Exodus, as opposed to the skin shields of today that fit close to the hull." He pauses... "... break the shields. And it is important to know that the pressure suit included with the ship is able to remotely activate and deactivate the shields and other ship systems. Good luck."`

Reply:
That does sound better. Thanks!